### DISCUSSION TITLE
自我介绍 | 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#253] [General]
# 自我介绍 | 愧怍的小站

愧怍的自我介绍

https://kuizuo.cn/about/

**Comment by chaseFunny** [comment]
你好，加油！

**Reply by kuizuo** [reply]
一同加油 💪

**Comment by Jinzedev** [comment]
膜拜一下佬

**Comment by l1725821492** [comment]
博客风格看着很舒服

**Comment by QiYongchuan** [comment]
你好大佬！

**Comment by chyun** [comment]
try

**Comment by kkkh** [comment]
博客非常好

**Comment by codergcy** [comment]
太牛了

**Comment by Skywt2003** [comment]
好漂亮的博客！😁

**Comment by lv123123long** [comment]
有关手指旋转的都能够杂耍一番。= 想看看（哈哈哈哈）

**Reply by kuizuo** [reply]
抽空把以前录制的视频放到这个 blog 上，献丑了🤭

**Comment by jiangyue95** [comment]
加油，x离职看到的。好厉害啊，我努力转码中。

**Reply by kuizuo** [reply]
望能在代码世界中找寻人生真正的意义💝

**Comment by pengweizhong** [comment]
你好，国内博客可以放国外社交媒体的链接吗？我看到你已经备案了

**Comment by Casta-mere** [comment]
大佬，怎么能申请添加一个友链呀

**Comment by mx-pai** [comment]
加油！！

**Comment by myysophia** [comment]
8年运维老兵，准备借助AI加入独立开发者。很高兴遇见你

**Comment by ghost** [comment]
Google查 docusaurus 相关资料时发现了博主，博主的魔改版docusaurus太棒了，连续读了好多博主的文章，是真大佬！

**Comment by PeterChen1997** [comment]
很赞！读完了最近几篇文章，收获很多好工具，感谢分享！也欢迎加友链~


title: '龙场茶室'
description: '一只练习时长五年半的 Web 开发练习生'
website: 'https://blog.peterchen97.cn/'
avatar: 'https://blog.peterchen97.cn/favicon.svg'

**Comment by ouyangjin-etn** [comment]
博主太牛了~

**Comment by zamlia** [comment]
博主能加一个友链嘛

title=咱们俩の博客
url=https://blog.zamlia.com
avatar=http://oss.zamlia.com/project/we-blog/images/avatar.jpg
description=勇往直前

**Comment by EL233** [comment]
加油！！

**Comment by njczy** [comment]
你好啊

**Comment by li-zhenyu** [comment]
愧怍君你好！最近我在做自己的个人站点，参考了这里的主题魔改系列教程。感谢！
另：闽公网安备35021102000847号所备案的域名`kzcode.cn`现在被其他人抢占，并且用作不良用途。为了避免风险，请您留意。

**Comment by coder-lzh** [comment]
你好  求个友链
{
  "title": "敲代码的卡卡罗特",
  "desc": "灵魂和身体都要在路上",
  "icon": "https://blog.share888.top/favicon.ico",
  "link": "https://blog.share888.top/"
}
